_apps/net.ser1.forage.md

```diff
@@ -1,20 +1,23 @@
 ---
 appid: net.ser1.forage
 title: Forage
-home: https://www.ser1.net
+home: https://sr.ht/~ser/forage
 icon: https://hg.sr.ht/~ser/forage/raw/assets/shopping_cart-blue.png
 screenshots:
 - url: https://hg.sr.ht/~ser/forage/raw/assets/AndroidScreenshot.jpg
 
-date: 2022-01-30 21:57:00-000
+date: 2022-03-21T13:20:24Z
```

Comment:
I don't think we should change this after the application gets added. This is just used to control the order that new additions are displayed on the site.